index.html

```diff
@@ -173,6 +173,44 @@ <h2>Calls for implementation and feedback</h2>
+
+In all possible cases, withholding consensus should take the form of a constraint. Blocks happen but should be avoided.
+
+<p>When a champion proposes a feature for advancement, a delegate may pose additional constraints and ask that the champion incorporate them into the proposal. The delegate should expect to spend time working with the champion to help flesh out the constraint and learn about other constraints that went into the proposal.
+
+<p>For features advancing from Stage 1 to Stage 2, a delegate may pose a constraint as necessary for advancement to Stage 3. In this situation, the delegate should expect to work with the champion and other delegates during Stage 2 to incorporate their constraint into the solution. In general, this is a good time to pose new constraints.
+
+<p>For features advancing to Stage 3, a delegate may ask that the champion work with them to incorporate a new constraint as a condition of advancement to Stage 3. In particular, given that consensus on Stage 3 means "the solution is complete", i.e., all open design issues havee beeen resolved including anticipated implementation and ecosystem compatibility issues, all TC39 participants should validate the design of proposals they care about before granting Stage 3 consensus.
+
+<p>Delegates should openly give feedback on proposals, especially when they have significant concerns, and especially for a proposal for stage advancement where the concern is relevant to the stage. Even if a Delegate does not intend to withhold consensus they should raise their concerns for general discussion.
+
+<p>A delegate may also request additional time to consider the proposal.
+
+<p>Conditional advancement is time-limited, giving the person raising the concern time to discuss with the champions and authors about their concerns. If a proposal has a conditional advancement, an issue must be opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.
+
+<p>The committee must record a good description of why a proposal did not advance, both in the meeting notes and within an issue in the proposal's tracker, but not limited to those. So that in future meetings and future proposals we can learn from failed proposals, and have a good record of why they failed.
+
+<p>In order to ensure that the committee is able to do it's work effectively, there are restrictions to witholding consensus.
+
+<p>The ability to withhold consensus for reaching stage 1, 2 and 3 is not limited but constraints introduced should be weighed according to their stage. Proposals which have fulfilled the acceptance criteria may be blocked from advancement for any well stated reason.
```

Comment:
```suggestion
<p>The ability to withhold consensus for reaching stage 1, 2, and 3 is not limited but constraints introduced should be weighed according to their stage. Proposals which have fulfilled the acceptance criteria may be blocked from advancement for any well-stated reason.
```

Reply:
good point. I used "weighed" as a short hand for "should be used in the appropriate stage", but really this isn't about constraints themselves, but withholding consensus. Maybe this should be changed to "withholding consensus should be used appropriately with the given stage"?

Reply:
I'm wondering if it might make sense to simply delete this paragraph. In practice, I believe we have been talking on a case-by-case basis about whether withholding consensus is appropriate for a particular stage, and I'm not sure if we need to change that practice.

Reply:
That makes sense, I have removed that sentence and reworded things a little bit.